recipes/newrelic/infrastructure/super-agent/logs/linux-logs.yml

```diff
@@ -0,0 +1,271 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: logs-super-agent-integration
+displayName: Logs Integration
+description: New Relic install recipe for Logging in super-agent
+repository: https://github.com/newrelic/infrastructure-agent/tree/master/assets/examples/logging
+
+# dependencies:
+#  - newrelic-super-agent
```

Comment:
This is commented because the newrelic-super-agent recipe is still not published, but it will need to be set once everything is published.